app/controllers/users_controller.rb

```diff
@@ -13,11 +13,16 @@ def index
       organizing: 'Helping as an Organizer'
     }
     @users = User.ordered(params[:sort],params[:direction])
-        .group('users.id').with_all_associations_joined
-        .page(params[:page])
+      .with_all_associations_joined.page(params[:page])
+    #.group('users.id').with_all_associations_joined
+      #  .total_pages
```

Comment:
I guess in this case, if you still need it you can look 'm up on GitHub, right? :heart: the beauty of GitHub!